robottelo/config/__init__.py

```diff
@@ -90,6 +90,19 @@ def get_url():
     return urlunsplit((scheme, hostname, '', '', ''))
 
 
+def default_nailgun_config():
```

Comment:
Can you please explain the rationale for adding a new function for this instead of passing the admin credentials to `user_nailgun_config`?

Reply:
yes, its indeed possible, but I just wanted to keep it separate from existing user_nailgun_config, as similar to what we had in nailgun. entity_mixin and its using `get_credentials` func to get creds so we don't need to pass and import settings just for ssh username and passwd in the tests

Reply:
I'm not totally convinced that adding a function with very similar functionality to an existing function is preferable to importing settings in a test module, especially since https://github.com/SatelliteQE/robottelo/pull/9936 suggests that `user_nailgun_config` can be used with both admin and non-admin users. I won't block the PR based on that, though.

However, I do think the name of the function and the docstring should make it clear why the new function exists. I would suggest changing the name of the function to `admin_server_config`, since it is designed specifically to return a `ServerConfig` object with the admin credentials. In the docstring, I would remove the reference to `:param user:` since the function has no parameters, and I would also make it clear in the docstring that the config file will be constructed from the admin credentials.

What do you think about these suggestions?

Reply:
@synkd Good point, renamed the func and updated the docstrings